src/components/AddToBasketButton/AddToBasketButton.js

```diff
@@ -74,13 +75,15 @@ class AddToBasketButton extends React.Component {
   render() {
     const { itemExistsInBasket } = this.state;
     const { addLabel, buttonProps, disabled, item, removeLabel } = this.props;
+    const coverage = JSON.stringify(get(item, '_object.coverage[0]', '-'));
```

Comment:
You might want to use a different default than `-` here, since you `JSON.parse` outside of a try-catch block in the tests, so if the eresource doesn't have any coverage, the test will crash and fail.